packages/repeat/src/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Repeats the input text number of times.
+ * @param {String} [text] The input text.
+ * @returns {Integer} times How many times shall we repeat the text, default to 1
+ * @param {String} glue The glue for the output, default to ''
```

Comment:
I love the naming here 🗡

Reply:
I like your solution, it's more elegant but is a little slower.  
I run a local performance test and when I try 100k repeat you can see the difference of execution.

Reply:
That's awesome @lazav94, I put mine as a point of reference and just exposing an alternative :)